Makefile

```diff
@@ -156,7 +156,7 @@ test-go-generate: ## Run go generate against test code.
 
 .PHONY: fmt
 fmt: ## Run go fmt against code.
-	$(GOFMT) -l -w -s $$(git ls-files --exclude-standard | grep "\.go$$")
+	$(GOFMT) -l -w -s $$(git ls-files --exclude-standard | grep "\.go$$" | grep -v $(GENERATED_CLIENT_PKG))
```

Comment:
`git ls-files --exclude=$(GENERATED_CLIENT_PKG)`，can this command achieve the same effect?

Reply:
<img width="476" alt="image" src="https://github.com/apecloud/kubeblocks/assets/111829739/dd5f8389-6723-4f5c-a784-de1bcc8d6b54">
the `exclude` option is for **untracked** files. But  dir `pkg/client` is tracked.